packages/payments-plugin/e2e/mollie-dev-server.ts

```diff
@@ -0,0 +1,90 @@
+import { AdminUiPlugin } from '@vendure/admin-ui-plugin';
+import { ChannelService, DefaultLogger, Logger, LogLevel, mergeConfig, PaymentMethodService } from '@vendure/core';
+import { createTestEnvironment, registerInitializer, SqljsInitializer, testConfig } from '@vendure/testing';
+import gql from 'graphql-tag';
+import localtunnel from 'localtunnel';
+import path from 'path';
+
+import { initialData } from '../../../e2e-common/e2e-initial-data';
+import { MolliePlugin } from '../package/mollie';
+import { molliePaymentHandler } from '../package/mollie/mollie.handler';
+
+import { CREATE_PAYMENT_METHOD } from './graphql/admin-queries';
+import { CreatePaymentMethod } from './graphql/generated-admin-types';
+import { AddItemToOrder } from './graphql/generated-shop-types';
+import { ADD_ITEM_TO_ORDER } from './graphql/shop-queries';
+import { CREATE_MOLLIE_PAYMENT_INTENT, setShipping } from './payment-helpers';
+
+/**
+ * This should only be used to locally test the Mollie payment plugin
```

Comment:
Do we want this file in GIT? This is meant for contributors (as mentioned in the readme). It's quite the hassle to setup a local env that actually communicates with the live Mollie env.

This dev-server only requires a Mollie apiKey and creates ready to test orders.

Reply:
Yes for sure, keep this in the repo!